zerver/middleware.py

```diff
@@ -186,6 +188,16 @@ def write_log_line(
                 f" (md: {format_timedelta(markdown_time_delta)}/{markdown_count_delta})"
             )
 
+    message_summary_output = ""
+    if "message_summary_time_start" in log_data:
+        message_summary_time_delta = (
+            get_message_summary_time() - log_data["message_summary_time_start"]
+        )
+
+        if message_summary_time_delta > 0.005:
+            # We only do 1 API call here, so logging that is not important here.
```

Comment:
I don't think we're necessarily guaranteed to always only do one query in a give code path, so I would probably have designed the logic to support logging multiple queries.

Reply:
Added!

2025-02-05 04:03:50.884 INFO [zr] 127.0.0.1       GET     200  2.4s (ai: 2.4s/1.0) (db: 36ms/5q) /json/messages/summary (32@zulip via Mozilla)